### PR TITLE
Fix the rules

### PR DESCRIPTION
**Pull request by nabijaczleweli**
As per http://chat.stackoverflow.com/transcript/message/25631969#25631969

### CODE REVIEW

**Comment by no-longer-on-githu-b** [comment]
:+1:

**Comment by garlique** [comment]
:+1:

**Comment by TonyTheLion** [comment]
+1

File: _the_law/the-rules.md [hunk 1]
```diff
@@ -1,129 +1,4 @@
...
-Shortcut                 | Function
------------------------- | --------
-<kbd>Enter</kbd>         | Send message, save edit
-<kbd>Shift+Enter</kbd>   | Add a newline to the message
-<kbd>↑</kbd>             | Enter edit mode (if message is single line and you have editable messages), go to previous message
-<kbd>↓</kbd>             | In edit mode, go to next message
-<kbd>Escape</kbd>        | Exit edit mode
-<kbd>Ctrl+K</kbd>        | Toggle fixed font
-<kbd>Tab</kbd>           | After typing `@`, choose between autocompleted nicks
+# COCK
```
Comment (sehe) [inline _the_law/the-rules.md:4]: I think I can agree with the general gist of the changes, but I'd remove this line.

Comment (nabijaczleweli) [inline _the_law/the-rules.md:4]: I hereby declare @sehe not worthy of participating in this discussion

Comment (garlique) [inline _the_law/the-rules.md:4]: I would like to take this opportunity to discuss the immediate transition of power over the Lounge to one Elim Garak, a loyal citizen of the Cardassian Union and the right hand of the head of the Obsidian Order.

Comment (nabijaczleweli) [inline _the_law/the-rules.md:4]: :+1:

**Comment by TheCatPlusPlus** [comment]
Funny.

**Pull request closed without merging**